apps/web/src/components/ExplorerDetails.tsx

```diff
@@ -35,7 +36,19 @@ function ExplorerDetailsItem({
 export function ExplorerDetails() {
   const { data: syncStateData } = api.syncState.getState.useQuery();
   const { data: blobStoragesState } = api.blobStoragesState.getState.useQuery();
+  const { data: latestBlock } = api.block.getLatestBlock.useQuery();
+
   const explorerDetailsItems: ExplorerDetailsItemProps[] = [
+    {
+      name: "Blob gas price",
+      icon: <Gas className="h-4 w-4" />,
+      value: latestBlock ? (
+        <div>
+          {latestBlock.blobGasPrice.toString()}
```

Comment:
We have a [component](https://github.com/Blobscan/blobscan/blob/next/apps/web/src/components/Displays/EtherUnitDisplay.tsx) to display ether units. Let's use it and fix the amount to `Gwei`